Jenkinsfile

```diff
@@ -15,8 +15,12 @@ pipeline {
     timeout(time: 30, unit: 'MINUTES')
   }
   environment {
-    /** See https://issues.jenkins-ci.org/browse/JENKINS-42771 - we'd like to expand this out into multi-line concatenations */
-    PYTEST_ADDOPTS = "-n=10 --tb=short --color=yes --driver=SauceLabs --variables=capabilities.json"
+    PYTEST_ADDOPTS =
+    "-n=10 " +
```

Comment:
Perfectly fine nit, and I did it correctly in fxapom, so I'll do it here and elsewhere before merging.